src/server.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const app = express();
+// const cors = require("cors");
+
+const { connectDatabase } = require("./utils/db");
+const { Environment } = require("./utils/environment");
+
+const port = Environment.getPort()
+const nodeEnv = Environment.getNodeEnv()
+
+if (nodeEnv === "development") {
+  require("dotenv").config();
+}
+
+connectDatabase()
+  .then(() => {
+    console.log("connectDatabase() success");
+  })
+  .catch((error) => {
+    console.error("connectDatabase() error:", error);
+  });
+
+
+/**
+ * Set view engine
+ */
+app.set("view engine", "ejs");
+
+
+/**
+ * Use middleware
+ */
+app.use(express.static("public"));
+app.use(express.urlencoded({ extended: true }));
+// app.use(express.json());
+// app.use(cors());
```

Comment:
This middleware is not necessary for the app in its current state but I didn't remove it incase you plan to make JSON endpoints.